Documentation/Storage-Configuration/Block-Storage-RBD/block-storage.md

```diff
@@ -197,3 +199,32 @@ The erasure coded pool must be set as the `dataPool` parameter in
 The erasure coded pool must be set as the `dataBlockPool` parameter in
 [`storageclass-ec.yaml`](https://github.com/rook/rook/blob/master/deploy/examples/flex/storage-class-ec.yaml). It is used for
 the data of the RBD images.
+
+### Recover RBD RWO volume in case of Node Loss
+
+!!! Note
+    The minimum required Kubernetes version require for this to support is v1.26 and assuming [CSI-add-ons](https://rook.github.io/docs/rook/latest/Storage-Configuration/Ceph-CSI/ceph-csi-drivers/?h=csiaddons#csi-addons-controller) CRD is deployed.
+
```

Comment:
Let's document the command for installing the CSI addons. Then they can follow the instructions directly from this guide, instead of following a link to the CSI addons. It seems simple to include here.

Reply:
added and while adding the steps I realise to enable this feature by default once user has the csi addons crd, we also need to enable `csi-addons` side which I think is missing part for automating whole process. I'm thinking of two approach here:

1. Enable `csi-addons` by default
2. Enable `csi-addons` when tainit is added and `csi-addons` crds are present. In this case, I'll add code to wait if all rbd csi-pods are running state

@travisn @Madhu-1 let me know if you have other approach or suggestion

Reply:
Also, do you want to modify the design doc too?

Reply:
That is reason i suggested disabling it by default and enable only if user wants. RBD csi driver might be enabled or might not be enabled by default.  Lets document the steps to enable the node watcher if user wants

Reply:
@travisn ^

Reply:
Do we need the csi addons sidecar enabled for this scenario? Or is the node watcher controller started with the above manifests?

Reply:
we need csi-addons sidecar to be enabled

Reply:
Honestly, these steps are complicated for the user to enable this feature. This is a good reminder that when creating a new feature it's good to start with the user documentation so we can see if it's simple enough.

IMO we really need to automatically enable the CSI addons sidecar if the CRDs are found. That would remove this step of enabling the setting. The user could choose to override it if they want to disable it, but at least the default should be to enable it with the CRDs.

The CSI addons is really feeling like a heavyweight solution to the node loss. Rook could easily just blocklist and unblocklist the nodes directly without using the CRD. I wonder if we should provide that implementation in the future if there is feedback that CSI addons are too much overhead for the users. I do still like that the blocklisting is tracked with a CRD so the admin can get status on blocklisting. Let's not implement the direct blocklisting now, but let's see about the feedback...

Reply:
I would like apologize for missing this requirement, it slipped my mind since the option was already in operator yaml.

We can definitely look for other options in future

Reply:
I don't want to rush the pr mentioned below or maybe even close that depending on what we decide. I just created what I had in mind before I forget since I'll be off sometime next week. 

#12487